frontend/src/app/testQueue/TestQueue.tsx

```diff
@@ -43,7 +44,7 @@ const emptyQueueMessage = (
 );
 
 export const queueQuery = gql`
-  query GetFacilityQueue($facilityId: ID!) {
```

Comment:
I didn't want to mess with this file too much because I only touch it for some css styling issues. Better to leave this migration to be bundle with a ticket that allows to throughly test this component.